src/engine/BMInterface.php

```diff
@@ -195,7 +196,7 @@ public function load_api_game_data($playerId, $gameId, $logEntryLimit) {
         return NULL;
     }
 
-       // Can the active player edit the most recent chat entry in this game?
+    // Can the active player edit the most recent chat entry in this game?
```

Comment:
Can you skip this change?  It'll cause a merge conflict with #2058.  If you can't, no big deal, i'll handle the merge conflict.